ADVANCED.md

```diff
@@ -1,45 +0,0 @@
-# Advanced Use-Cases
```

Comment:
Do we want to scrap this without replacement?

Reply:
no, it is here: so for most contents I copied them over into other parts of the docu. The essence of this guide should be briefly summarized in https://github.com/sse-secure-systems/connaisseur/blob/docs/documentation-2.0/docs/validators/notaryv1.md#using-azure-container-registry
so copied over for now and will have to be melted into the new page by the author